fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/RelOpValue.java

```diff
@@ -734,7 +743,10 @@ private enum UnaryPhysicalOperator {
         IS_NOT_NULL_FI(Comparisons.Type.NOT_NULL, Type.TypeCode.FLOAT, Objects::nonNull),
         IS_NOT_NULL_DI(Comparisons.Type.NOT_NULL, Type.TypeCode.DOUBLE, Objects::nonNull),
         IS_NOT_NULL_SS(Comparisons.Type.NOT_NULL, Type.TypeCode.STRING, Objects::nonNull),
-        IS_NOT_NULL_BI(Comparisons.Type.NOT_NULL, Type.TypeCode.BOOLEAN, Objects::nonNull);
+        IS_NOT_NULL_BI(Comparisons.Type.NOT_NULL, Type.TypeCode.BOOLEAN, Objects::nonNull),
```

Comment:
Not a comment to you but I have not clue what the last prefix represents shouldn't this always be just representing the input type like in your case? I mean what does the `I` in `IS_NOT_NULL_FI` stand for?

Reply:
I have no clue either. I also don't know why all the other ones have `I` as a suffix, but strings have `S`.